BibBuddy/app/src/main/java/de/bibbuddy/LibraryFragment.java

```diff
@@ -149,6 +162,64 @@ public void onRequestPermissionsResult(int requestCode,
     }
   }
 
+  private void createBibFile(String folderName, String fileName) {
+    try {
+      String rootPath = Environment.getExternalStorageDirectory()
+          .getAbsolutePath() + "/" + folderName + "/";
+      File root = new File(rootPath);
+
+      if (!root.exists()) {
+        root.mkdirs();
+      }
+
+      File file = new File(rootPath + fileName + ".bib");
+      if (file.exists()) {
+        file.delete();
+      }
+      file.createNewFile();
+
+      FileOutputStream out = new FileOutputStream(file);
+      out.flush();
+      out.close();
+
+      Toast.makeText(context, "File exported in: " + '\n'
+          + rootPath + fileName + ".bib", Toast.LENGTH_LONG).show();
+
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+  }
+
+  private void writeBibFile(String folderName, String fileName) {
+    try {
+      File dir = new File(Environment.getExternalStorageDirectory()
+          .getAbsolutePath() + "/" + folderName + "/");
+
+      if (!dir.exists()) {
+        dir.mkdirs();
+      }
+
+      File statText = new File(Environment.getExternalStorageDirectory()
+          + "/" + folderName + "/" + fileName + ".bib");
+
+      FileOutputStream fos = new FileOutputStream(statText);
+      OutputStreamWriter osw = new OutputStreamWriter(fos);
+      Writer fileWriter = new BufferedWriter(osw);
+
+      fileWriter.write("@book{hawking1988," + '\n'
```

Comment:
If I remember right, the expected behaviour agreed upon in last team meeting was that all books present in the shelf are exported. This here, if I see this right, only exports an example? Why is that - wouldn't it be better to merge the finished feature instead of an example file? 
Other opinions @SarahKurek @claudia3?

Reply:
Yes, this is the title of this pull request. #83 Export an example .bib file to device storage

Reply:
Well yes I read that :D But does it make sense to implement that for the second release if it's just an example? I feel like waiting a couple of days more and then actually having the full feature would be a better option?

Reply:
Hi! You asked me to review the bibtex format here in the team meeting. I think it should mostly work, however some fields are not correct.
* Not quite sure why the "n" of Stefan is in separate {}?
* year has to be in {} as well :)
* the correct fieldname for a note is "annote"
* the note itself has to be in {} as well
* Why is the id "hawking1988" when the author is called "Sweig" and the year is "1942"? Should be "sweig1942" instead :)

So your example correct would be:

@book{sweig1942
title = {The impossible book},
author = {Stefan Sweig},
year = {1942},
publisher = {Dead Poet Society},
anote = {some note example}
}

Hope I helped! :)


See this as reference:

@misc{brownlee_gentle_2019,
    title = {A Gentle Introduction to Computer Vision},
    url = {https://machinelearningmastery.com/what-is-computer-vision/},
abstract = {Computer Vision, often abbreviated as CV, is defined as a field of study that seeks to develop techniques to help computers “see” and understand the content of digital images such as photographs and videos. The problem of computer vision appears simple because it is trivially solved by people, even very young children. Nevertheless, it largely […]},
    language = {en-US},
    urldate = {2021-02-14},
    journal = {Machine Learning Mastery},
    author = {Brownlee, Jason},
    month = mar,
    year = {2019},
    annote = {Das hier ist eine Textnotiz!},
    file = {Snapshot:/das/hier/ist/eine/Datei.html:text/html},
}